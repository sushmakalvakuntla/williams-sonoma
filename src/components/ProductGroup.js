import React from 'react';
import Product from './Product';
import { Col } from 'react-bootstrap';

const ProductGroup = ({ data }) => {
  return data?.groups.map((product, index) => (
    <Col lg={4} sm={12} key={index.toString()}>
      <Product product={product} />
    </Col>
  ));
};

export default ProductGroup;
