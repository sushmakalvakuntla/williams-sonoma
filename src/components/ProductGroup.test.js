import React from 'react';
import { shallow } from 'enzyme';
import ProductGroup from './ProductGroup';

it('renders ProductGroup correctly', () => {
  const component = shallow(<ProductGroup />);
  expect(component).toMatchSnapshot();
});
