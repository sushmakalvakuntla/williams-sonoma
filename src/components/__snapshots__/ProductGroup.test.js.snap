// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ProductGroup correctly 1`] = `""`;
